dbt_dry_run/test/node_runner/test_model_runner.py

```diff
@@ -13,6 +13,10 @@
 
 VIEW_CREATION_SQL = "CREATE OR REPLACE VIEW"
 
+DBT_MAX_PARTITION_DECLARATION = (
```

Comment:
This isn't reused across the test suite so it can just be a local variable in `test_model_with_dbt_max_partition_variable_replaces_it`